source/bank_01.asm

```diff
@@ -31,6 +32,38 @@ level_name_textbox_clear:
     jp nz, .loop
     ret
 
+SECTION "calc_town_map_level_index", ROMX[$61FA], BANK[$01]
+calc_town_map_level_index::
+    ld a, [w_town_map_zone]
+    cp a, 10
```

Comment:
```suggestion
    cp 10
```

Reply:
Also, if the zones have indices, please create constants for them - they have proper names in-game.